src/patternfly/components/Accordion/accordion-item.hbs

```diff
@@ -0,0 +1,6 @@
+<li class="pf-c-accordion__item{{#if accordion-item--IsExpanded}} pf-m-expanded{{/if}}{{#if accordion-item--modifier}} {{accordion-item--modifier}}{{/if}}"
```

Comment:
The `aria-expanded` attribute should always be placed on the interactive element. So in this case, it should be moved from the `<li>` to the `button.pf-c-accordion__item-button`.